### PR TITLE
prepare v2.0.0 release

### PR DESCRIPTION
**Pull request by lgfa29**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,7 +37,8 @@ BUG FIXES:
 * resources/nomad_job: fix a bug that prevented deployments for jobs in namespaces other than `default` from being monitored. ([#347](https://github.com/hashicorp/terraform-provider-nomad/pull/347))
 * resource/nomad_job: fix a bug that could result in unnecessary plan diffs from irrelevant changes. ([#356](https://github.com/hashicorp/terraform-provider-nomad/pull/356))
 * resource/nomad_volume: fix a bug that caused `capacity_min` and `capacity_max` to be mandatory. ([#363](https://github.com/hashicorp/terraform-provider-nomad/pull/363))
-* resource/nomad_volume and resource/nomad_external_volume: fix a bug where `topology_request` was not persisted to state. ([#342](https://github.com/hashicorp/terraform-provider-nomad/pull/342)
+* resource/nomad_volume: fix a bug where `topology_request` was not persisted to state. ([#342](https://github.com/hashicorp/terraform-provider-nomad/pull/342)
+* resource/nomad_external_volume: fix a bug where `topology_request` was not persisted to state. ([#342](https://github.com/hashicorp/terraform-provider-nomad/pull/342)
```
Comment (gulducat) [inline CHANGELOG.md:41]:
```suggestion
* resource/nomad_external_volume: fix a bug where `topology_request` was not persisted to state. ([#342](https://github.com/hashicorp/terraform-provider-nomad/pull/342))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,12 +18,13 @@ DEPRECATIONS:
...
 * provider: add `skip_verify` configuration to skip TLS verification ([#319](https://github.com/hashicorp/terraform-provider-nomad/pull/319))
 * provider: update Go to 1.21.0 ([#369](https://github.com/hashicorp/terraform-provider-nomad/pull/369))
 * data source/nomad_namespace: add `node_pool_config` attribute ([#355](https://github.com/hashicorp/terraform-provider-nomad/pull/355))
 * resource/nomad_acl_policy: add support for `job_acl` ([#314](https://github.com/hashicorp/terraform-provider-nomad/pull/314))
-* resource/nomad_csi_volume and resource/nomad_csi_volume_registration: add support to import existing volumes. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359)]
+* resource/nomad_csi_volume: add support to import existing volumes. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359)]
+* resource/nomad_csi_volume_registration: add support to import existing volumes. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359)]
 * resource/nomad_job: add support to import existing jobs. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359)]
```
Comment (gulducat) [inline CHANGELOG.md:28]:
```suggestion
* resource/nomad_csi_volume: add support to import existing volumes. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359))
* resource/nomad_csi_volume_registration: add support to import existing volumes. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359))
* resource/nomad_job: add support to import existing jobs. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359))
```

**Review by gulducat — APPROVED** [review]
I'm just nit-picking the variety of closing parentheses really :yum:

**Comment by lgfa29** [comment]
> I'm just nit-picking the variety of closing parentheses really 😋

Ugh...I hate having to do this by hand. Thanks!

**Pull request merged**